machineProblem.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# -*- coding: utf8 -*-
+
+# unified interface introduced in branch issues/framework
+
+import sys
+import tkinter
+from tkinter import messagebox
+from tkinter import font
+
+LARGE_FONT= ("Verdana", 12)
+
+class SeaofBTCapp(tkinter.Tk):
+
+    def __init__(self, *args, **kwargs):
+    
+        tkinter.Tk.__init__(self, *args, **kwargs)
+        container = tkinter.Frame(self)
+        
+        container.pack(side="top", fill="both", expand = True)
+
+        container.grid_rowconfigure(0, weight=1)
+        container.grid_columnconfigure(0, weight=1)
+
+        self.frames = {}
+
+        for F in (StartPage, yourName, highscores):
```

Comment:
append the class(frame/page) name to this array for every class added

also, class names start with an Upper case, pls fix that